src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/util/PaginationUtils.kt

```diff
@@ -40,3 +40,29 @@ fun <T>getMetadata(response: Page<T>, page: Int?): PaginationMetadata? {
     null
   }
 }
+
+fun getIndices(page: Int, taskCount: Int): Pair<Int, Int> {
```

Comment:
Might be a good idea to get some unit testing in here, just to be sure it copes with things like uneven counts and things like that. It'll also document the behaviour for future us so the behaviour's not buried in the integration tests.